packages/browser/test/e2e/server.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+const express = require('express');
+
+const app = express();
+const port = process.env.PORT || 8080;
```

Comment:
nit:
`process.env.PORT` will always be empty here and given that there is already more then one instance of `"8080"` I recommend using it from `playwright.config.js` like

`config.webServer.port`

Reply:
even better if there is a native way to use it from playwright.
Something like (psudo code)
```javascript
import config from "playwright"

const port = config.port
```

Reply:
Moving the server to `fixtures` now, we can also define and control `port` as a fixture as well (which is probably the most reliable / native way).